pyat/developers.rst

```diff
@@ -105,6 +105,12 @@ To do this, we use the continuous integration service Github Actions.
 When a tag of the form pyat-* is pushed to Github, wheels for each
 supported platform will be built and automatically uploaded as an 'artifact'.
 
+The version number is of the form of three dot separated numbers:
+``major.minor.patch``. Major versions include major new functionalities but
```

Comment:
Is this an attempt to describe Semantic Versioning? If so:

- I would recommend using the phrase 'Semantic Versioning' and linking to the associated specification (https://semver.org/)
- You should correct your definition of Major versions; it does not need to include new functionality (even minor) - it only reflects breaking changes. Majority of new development (and new features) should be done on minor increments, as we would prefer to maintain compatibility where possible

If you think there are aspects of the official semver spec that are difficult for us to implement, it would be useful to discuss (here or elsewhere).

Reply:
The paragraph is updated.